docs/modules/ROOT/partials/base/scalar-expressions.adoc

```diff
@@ -56,8 +59,10 @@ SELECT PROBABILITY OF VAR Apogee_km > Apogee_km UNDER military_model
 ====
 [source,iql]
 ----
-SELECT PROBABILITY DENSITY OF VAR Apogee_km < 1000
-UNDER model CONDITIONED BY VAR Users = "Military"
+SELECT
+  PROBABILITY DENSITY OF VAR Apogee_km < 1000
```

Comment:
since we're fixing this:   `PROBABILITY DENSITY OF VAR Apogee_km < 1000` will crash because `PROBABILITY DENSITY` takes a density event (i.e. involving equality). `VAR Apogee_km < 1000` above is a distribution event.